src/models/request.js

```diff
@@ -14,7 +14,7 @@ export default (sequelize, DataTypes) => {
     },
     status: {
       type: DataTypes.STRING,
-      allowNull: false,
+      defaultValue: 'Pending',
```

Comment:
@fegzycole I discovered that the status should `Open`, `Approved` or `Rejected`. Check [this story](https://www.pivotaltracker.com/story/show/167733837)

Reply:
Thank you @Ileriayo. We agreed as a team to use Pending, Approved or Rejected. That was what informed my setting the default value to Pending.

Reply:
Okay, I was thinking the decision was before we noticed the statuses on the PT board. Nevertheless, when we get to tasks that states default as `open` (implicitly), you may have to inform the various person's working on those tasks so that they do not bang their heads against the wall because of that status.

Besides the above, I'd say it's good for a merge @Oluwafayokemi.

Reply:
Great job man!
Good to go @Oluwafayokemi.